gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -288,6 +331,295 @@ gst_tensor_src_iio_init (GstTensorSrcIIO * self)
   return;
 }
 
+/**
+ * @brief check if device/trigger with the given name exists
+ * @return >=0 if OK, represents device/trigger number
+ *         -1  if returned with error
```

Comment:
the function return type is unsigned.

Reply:
Corrected here as well as at other places.